lib/cisco_node_utils/bgp_af.rb

```diff
@@ -264,8 +258,7 @@ def default_advertise_l2vpn_evpn
 
     # dampen_igp_metric
     def dampen_igp_metric
-      result = config_get('bgp_af', 'dampen_igp_metric', @get_args)
-      result ? result.first.to_i : nil
+      config_get('bgp_af', 'dampen_igp_metric', @get_args)
```

Comment:
Ah, I was mistaken.  auto_default causes `config_get` to return nil instead of the default, which works for this property.